bundles/customer-registration/composer.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "fond-of-oryx/customer-registration",
+  "description": "n/a",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Hyatt",
+      "email": "dev@jzit.de"
+    }
+  ],
+  "require": {
+    "php": ">=7.4",
+    "fond-of-oryx/customer-registration-extension": "^1.0.0-beta.1",
+    "fond-of-spryker/customer": "dev-2022-spryker-upgrade",
+    "spryker/application": "^3.0.0",
+    "symfony/security-core": "^v4.4.44"
```

Comment:
Are `fond-of-spryker/customer` and `symfony/security-core` necessary dependencies?

Reply:
first one is needed since we need our ref stuff. But I will check again